pkg/analyzer/analyzer.go

```diff
@@ -46,7 +46,7 @@ var coreAnalyzerMap = map[string]common.IAnalyzer{
 }
 
 var additionalAnalyzerMap = map[string]common.IAnalyzer{
-	"HorizontalPodAutoScaler": HpaAnalyzer{},
+	"HorizontalPodAutoscaler": HpaAnalyzer{},
```

Comment:
This would break `k8sgpt filters add`, maybe I should remove it?

Reply:
Are you saying because this is using v2 api it would break on default?

Reply:
I’m sorry, this wasn't clear. I mean that renaming the filter won’t align with the `active_filters` value used in an existing `k8sgpt.yaml` config file. This might need a config migration.

Reply:
I'll merge this in - can you please validate after we push the new version? (v0.4.0)

Reply:
Sure thing